Makefile

```diff
@@ -3,6 +3,7 @@ IMAGE_PREFIX      ?= kubernetes-helm
 SHORT_NAME        ?= tiller
 SHORT_NAME_RUDDER ?= rudder
 TARGETS           ?= darwin/amd64 linux/amd64 linux/386 linux/arm linux/arm64 linux/ppc64le windows/amd64
```

Comment:
I'm thinking it probably makes sense to merge your other PR with this change   #4190

Reply:
I guess this modification is a little bit more complex. In https://github.com/kubernetes/helm/pull/4190 I simply added an architecture